amundsen_application/static/js/components/common/Breadcrumb/index.tsx

```diff
@@ -23,17 +23,15 @@ export const Breadcrumb: React.SFC<BreadcrumbProps> = (props) => {
     path = '/';
     text = 'Home';
     if (props.searchTerm) {
-      path = `/search?searchTerm=${props.searchTerm}&selectedTab=table&pageIndex=0`
-      text = 'Search Results'
+      path = `/search?searchTerm=${props.searchTerm}&selectedTab=table&pageIndex=0`;
```

Comment:
Probably need to fix the hard-coded `selectedTab` and `pageIndex`. 

FYI: @ryanlieu

Reply:
@ttannis this was the other quality-of-life issue that I wanted to fix separately.